.eslintrc.json

```diff
@@ -13,6 +13,19 @@
       "error",
       "global"
     ],
-    "no-console": "off"
+    "no-console": "off",
+    "prefer-destructuring": [
+      "warn",
```

Comment:
Why warn and not error?

Reply:
I usually reserve the `error` level for actual errors and use `warn` for stylistic preferences. But I understand that there are downsides to it because these won't be caught in the pre-commit hook. I can change it if you prefer.

Reply:
I'd prefer only errors in the eslint config to make it actionable. warning are always ignored IMO.